src/bespokelabs/curator/request_processor/base_online_request_processor.py

```diff
@@ -124,8 +125,10 @@ def __init__(
         frequency_penalty: Optional[float] = None,
         max_requests_per_minute: Optional[int] = None,
         max_tokens_per_minute: Optional[int] = None,
+        require_all_responses: bool = False,
```

Comment:
Default should be True, otherwise users are in for a surprise!

Reply:
Changed it to None everywhere except for base_request_processor where it is True